usaspending_api/api_docs/api_documentation/README.md

```diff
@@ -0,0 +1,28 @@
+
+# USASpending API Documentation
```

Comment:
This seems redundant to https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_docs/markdown/api_tutorial.md.

I would update that with relevant v2 examples and account for the feedback below as well.

Reply:
removed

Reply:
Still seeing the file here, is this removal happening somewhere else?

Reply:
just commited